tests/query/stateless/test_lookup.py

```diff
@@ -18,12 +18,14 @@ def prepare(self):
         self.check_resp_succeeded(resp)
 
     def test_edge_index(self):
+        time.sleep(self.delay)
         resp = self.execute('USE nbaLookup')
         self.check_resp_succeeded(resp)
         resp = self.execute("CREATE EDGE like(likeness int)")
         self.check_resp_succeeded(resp)
         resp = self.execute("CREATE EDGE serve(start_year int, end_year int)")
         self.check_resp_succeeded(resp)
+        time.sleep(self.delay)
```

Comment:
why here need `sleep`

Reply:
Waiting meta client.

Reply:
> why here need `sleep`

Need to get `Edge` from MetaClient when create edge index, So need to wait for data synchronization between `MetaClient` and `MetaServer`.

Reply:
I think the create index can't depend on the schema from the cache. it can check in metad, why check it by the cache.

Reply:
> I think the create index can't depend on the schema from the cache. it can check in metad, why check it by the cache.

You are right, This logic has been improved via #316 .I think it's best to add to regression test quickly.  Then fix the delay problem when the other PR is merged.